### ISSUE TITLE
external subnets are not correctly assigned external_securitygroup_ids

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/terraform-aws-bigip-module#25]
External security groups are not correctly assigned to NICs when multiple external subnet and security groups exist.

## Current Behavior
NICs are assigned all security groups found in the parameter value. This is due to pulling value from var instead of the local mappings.

```
resource "aws_network_interface" "public" {
  count             = length(compact(local.external_public_private_ip_primary)) > 0 ? length(local.external_public_subnet_id) : 0
  subnet_id         = local.external_public_subnet_id[count.index]
  security_groups   = var.external_securitygroup_ids
```

As a result, all external subnet NICs get all security groups assigned found in var.external_securitygroup_ids

## Expected Behavior
Security group and subnet should be a 1 to 1 mapping. The below config should result in 5 subnets with 5 security groups. 

```
module "bigip" {
  source                     = "F5Networks/bigip-module/aws"
  external_subnet_ids        = [local.ext_subnets.external, local.ext_subnets.dmz1, local.ext_subnets.dmz2, local.ext_subnets.dmz3, local.ext_subnets.dmz4]
  external_securitygroup_ids = [local.ext_security_groups.external, local.ext_security_groups.dmz1, local.ext_security_groups.dmz2, local.ext_security_groups.dmz3, local.ext_security_groups.dmz4]
}
```

**Comment by JeffGiroux** [comment]
## Workaround
Code needs to be updated on each of the network interface blocks to use something similar to this...

external subnets with public IP = true
```
  security_groups   = [local.external_public_security_id[count.index]]
```

external subnet with public IP = false
```
  security_groups   = [local.external_private_security_id[count.index]]
```

And so on...

**Comment by KrithikaChidambaram** [comment]
This request is now being tracked internally with ID INFRAANO-859. Thanks!

**Comment by RavinderReddyF5** [comment]
@JeffGiroux fixed in v1.1.7, please verify
